src/main/java/org/folio/fqm/repository/DataRefreshRepository.java

```diff
@@ -81,6 +83,11 @@ public class DataRefreshRepository {
    * @return            True if refresh successful, false otherwise
    */
   public boolean refreshExchangeRates(String tenantId) {
+    if (!entityTypeInitializationService.isModFinanceInstalled()) {
+      log.info("mod-finance is not installed and entities using exchange rates are unavailable; skipping exchange rate refresh");
+      return true;
+    }
+
```

Comment:
This operation can scream into the void with a million 404s failing to fetch exchange rates if mod-finance is unavailable, but this felt like good manners.